dev-games/libsmacker/libsmacker-1.1.1.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils
+
+DESCRIPTION="A cross-platform C library for decoding .smk Smacker Video files."
+HOMEPAGE="https://libsmacker.sourceforge.net"
+
+SRC_URI="https://sourceforge.net/projects/libsmacker/files/libsmacker-1.1/libsmacker-1.1.1r35.tar.gz/download -> ${P}.tar.gz"
```

Comment:
Is this URL more or less stable between releases? It would be better to use variables with versions so this URL doesn't have to be touched every version bump in the future.

Reply:
I cant say. I think those variables just make it unreadable but will do it anyway.

Reply:
Without *some* variables, the URL needs to be updated on every version bump. E.g. 1.1.1 will be different on next version. `$(ver_cut)` etc lets to reuse existing SRC_URI between bumps

But yes, it shouldn't be replaced everywhere

Reply:
yeah - but replacing libsmacker-1.1 with ${PN}-1.1 will break on -1.2 is there a variable which can cut off the last part of version?
I dont understand how to use $(ver_cut)

Reply:
https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html#obtaining-version-substring-ver-cut

And don't replace with PN. Replacing with P is more useful than with PN

Reply:
thats what I thought - but @ajakk suggested it.

Reply:
Please reread my comment (in the other thread below)